components/tabs/stories/tabs.stories.js

```diff
@@ -105,38 +146,154 @@ export default {
 	},
 };
 
+/**
+ * Basic, default tab items should have a label for accessibility. If a label isn’t present, it must include an icon and becomes an icon-only tab item.
+ * 
+ * By default, tabs have a divider that spans across all tab items. This style works as a way to anchor them to the page. These types of tabs are best used at the top of a page, as a top-level navigation.
+ * 
+ */
 export const Default = TabsGroups.bind({});
 Default.args = {};
 
-// ********* VRT ONLY ********* //
-export const WithForcedColors = TabsGroups.bind({});
-WithForcedColors.tags = ["!autodocs", "!dev"];
-WithForcedColors.parameters = {
-	chromatic: {
-		forcedColors: "active",
-		modes: disableDefaultModes
-	},
-};
-
 // ********* DOCS ONLY ********* //
-export const Vertical = Template.bind({});
-Vertical.tags = ["!dev"];
+/**
+ * Vertical tabs should be used when horizontal space is more generous and when the list of sections is greater than can be presented to the user in a horizontal format. 
+ */
+export const Vertical = VerticalGroup.bind({});
 Vertical.args = {
 	orientation: "vertical",
 };
+Vertical.tags = ["!dev"];
 Vertical.parameters = {
-	chromatic: {
-		disableSnapshot: true,
-	},
+	chromatic: { disableSnapshot: true },
 };
 
-export const Overflow = Template.bind({});
-Overflow.tags = ["!dev"];
+export const VerticalRight = Template.bind({});
+VerticalRight.args = {
+	...Vertical.args, 
+	hasRightAlignedTabs: true,
+};
+VerticalRight.storyName = "Vertical right";
+VerticalRight.tags = ["!dev"];
+VerticalRight.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * When there are too many tabs to fit horizontally across the viewport, the tabs component can be displayed as a [quiet picker](/docs/components-picker--docs). The example below is non-functional.
+ * 
+ * When appropriate, you can use alternative methods of overflowing tabs such as horizontal scrolling.
```

Comment:
How do you feel about changing this to passive voice?

```suggestion
 * When appropriate, alternative methods of overflowing tabs such as horizontal scrolling can be used.
```

Reply:
Yes! 🙌  As this is another instance of me [copy/pasting,](https://spectrum.adobe.com/page/tabs/#Tab-overflow) should I be on the lookout to make sure any documentaiton is in the passive voice?

https://github.com/adobe/spectrum-css/pull/2925/commits/4e6c79f317bc95714d6532665af35d20d36e1f43